Libplanet/Action/AccountStateDeltaImpl.cs

```diff
@@ -180,6 +185,15 @@ public IAccountStateDelta BurnAsset(Address owner, FungibleAssetValue value)
             );
         }
 
+        [Pure]
+        private FungibleAssetValue GetBalance(
+            Address address,
+            Currency currency,
+            IImmutableDictionary<(Address, Currency), BigInteger> state) =>
```

Comment:
How about renaming as others like `fungibleAssets` or better? because I think the word `state` can cause confusion with `state` used in `GetState`, `updatedStates`.

Reply:
It would be better to have a different name like `balances`.  At least, the singular `state` should be a plural `states`.

Reply:
https://github.com/planetarium/libplanet/pull/1134/commits/22aa91a244839f71dfd6155a734d0151f0182d7f amended.